packages/zowe-explorer/src/extension.ts

```diff
@@ -430,11 +430,24 @@ function initDatasetProvider(context: vscode.ExtensionContext) {
             }
         })
     );
+
     context.subscriptions.push(vscode.commands.registerCommand("zowe.ds.renameDataSet", (node) => datasetProvider.rename(node)));
-    context.subscriptions.push(vscode.commands.registerCommand("zowe.ds.copyMember", (node) => dsActions.copyDataSet(node)));
-    context.subscriptions.push(vscode.commands.registerCommand("zowe.ds.copyDataSet", (node) => dsActions.copyDataSet(node)));
-    context.subscriptions.push(vscode.commands.registerCommand("zowe.ds.pasteMember", (node) => dsActions.pasteMember(node, datasetProvider)));
```

Comment:
Since we are in an LTS release, removing commands might be considered a breaking change.
We should implement this new behavior with new commands and deprecate the old ones.

Already in the middle of this 😋

Reply:
🤔 Removing the commands may not be a breaking change since they were never expose to the end user.
However, changing an exported function's signature is.
I'm happy to work on this in order to deprecate the copyDataSet method 😋